components/clinical/diagnosis-summary/src/organisms/diagnosis-summary.tsx

```diff
@@ -6,59 +6,128 @@ import { DateSummary } from '@ltht-react/type-summary'
 import Icon from '@ltht-react/icon'
 import { Button, ButtonProps } from '@ltht-react/button'
 
-import { BTN_COLOURS, SMALL_SCREEN_MAXIMUM_MEDIA_QUERY, WIDESCREEN_MINIMUM_MEDIA_QUERY } from '@ltht-react/styles'
+import {
+  BTN_COLOURS,
+  DESKTOP_MINIMUM_MEDIA_QUERY,
+  SMALL_SCREEN_MAXIMUM_MEDIA_QUERY,
+  TABLET_MINIMUM_MEDIA_QUERY,
+  WIDESCREEN_MINIMUM_MEDIA_QUERY,
+} from '@ltht-react/styles'
 import Title from '../atoms/diagnosis-title'
 import OnsetDateEstimated from '../atoms/diagnosis-onset-estimated'
 import Redacted from '../molecules/diagnosis-redacted'
 import DiagnosisDataSource from '../atoms/diagnosis-data-source'
 
 const StyledTitle = styled.div`
-  display: inline-block;
+  display: flex;
 `
+
 const StyledSummary = styled.div`
   display: flex;
   justify-content: center;
+  flex-direction: column;
 `
+
 const StyledDescription = styled.div`
   flex-grow: 1;
 `
-const StyledLeftContainer = styled.div`
+
+const StyledTitleAndDateContainer = styled.div`
   display: flex;
   flex-grow: 1;
-  flex-direction: column;
-  gap: 0.3rem;
+  flex-direction: row;
+  gap: 5px;
+
+  ${SMALL_SCREEN_MAXIMUM_MEDIA_QUERY} {
+    flex-flow: column nowrap;
+    align-items: flex-start;
+    div {
+      text-align: left;
+    }
+  }
+`
 
-  ${WIDESCREEN_MINIMUM_MEDIA_QUERY} {
+const StyledButtonAndTagContainer = styled.div`
+  display: flex;
+  flex-grow: 1;
+  flex-direction: row;
+  gap: 5px;
+  margin-top: 0.5rem;
+
+  ${SMALL_SCREEN_MAXIMUM_MEDIA_QUERY} {
     flex-direction: row;
+    justify-content: space-between;
   }
 `
-const StyledResponsiveContainer = styled.div`
+
+const StyledResponsiveButtonContainer = styled.div`
   display: flex;
-  margin: 0;
+  margin-right: auto;
   flex-flow: row wrap;
-  gap: 0.3rem;
-  align-content: center;
+  gap: 5px;
+  height: fit-content;
+  flex-grow: 1;
 
-  ${WIDESCREEN_MINIMUM_MEDIA_QUERY} {
-    margin: 0 10px 0 10px;
+  & > * {
+    height: 1.5rem;
+    padding: 0 5px;
+
+    ${TABLET_MINIMUM_MEDIA_QUERY} {
+      font-size: 0.9rem !important;
+    }
   }
 
   ${SMALL_SCREEN_MAXIMUM_MEDIA_QUERY} {
     margin: 0;
     flex-direction: column;
-    align-content: flex-start;
+
+    span {
+      width: 100%;
+      max-width: 10rem;
+    }
+  }
+`
+
+const StyledResponsiveTagContainer = styled.div`
+  display: flex;
+  margin-left: auto;
+  flex-flow: row wrap;
+  gap: 5px;
+  height: fit-content;
+  justify-content: flex-end;
+  flex-grow: 1;
+
+  & > * {
+    display: flex;
+    align-items: center;
+    height: 1.5rem;
+    margin: 0 !important;
+    font-size: 0.8rem !important;
+    padding: 0 10px;
+    justify-content: center;
+
+    ${TABLET_MINIMUM_MEDIA_QUERY},
```

Comment:
I think (_think_) you can get away with just `TABLET_MINIMUM_MEDIA_QUERY` here, although I do like how clear it is to read as a dev 👌